doc/rm/c_cpp_coding_style.md

```diff
@@ -173,6 +186,49 @@ This may change if we find cases where this initialization improves readability.
 When initializing an array, initializers *may* be designated when that makes the array more readable (e.g., lookup tables that are mostly zeroed). Mixing designated and undesignated initializers, or using nested initializers, is still
 forbidden.
 
+### Nonstandard Attributes
+
+The following nonstandard attributes may be used:
+*   `section(<name>)` to put a definition into a given object section.
+*   `weak` to make a symbol definition have weak linkage.
+*   `naked` to ensure a function has no prolog or epilog.
+*   `packed` to ensure a struct contains no padding.
```

Comment:
Please also add `interrupt` to this list.

Reply:
You probably also want "aligned" or whatever GCC calls its explicit alignment attribute.

Also, maybe include "alias"? It can be useful for making stuff like `memcpy` link without actually having to define `memcpy` (e.g. if we define `ot_memcpy` instead).